assets/styles/_variables.scss

```
@@ -8,23 +8,64 @@ $container-max-width: 1440px;
 $container-padding-desktop: 60px;
 $container-padding-tablet: 40px;
 $container-padding-mobile: 15px;
-$content-max-width: 600px;
+$content-max-width: 760px;
 $component-spacing-desktop: 120px;
 $component-spacing-tablet: 80px;
 $component-spacing-mobile: 40px;
 $gutter-width: 24px;
 
-$color-brand: #0d8eff;
-$color-accent: #f96417;
-$color-background: #fff;
-$color-contrast: #414751;
-$color-text: $color-contrast;
-$color-headline: #0b1016;
-$color-border: #d8d8d8;
+// Theming variables
+:root {
```

As `.themeReset` is supposed to be the same as `:root`, could we combine the selectors?

```suggestion
:root,
.themeReset {
```